components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -240,6 +240,20 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         }
     }
 
+    /**
+     * Migrate from a logged-in Firefox Account.
+     *
+     * Modifies the FirefoxAccount state.
+     *
+     * This performs network requests, and should not be used on the main thread.
+     */
+    fun migrateFromSessionToken(sessionToken: String, kSync: String, kXCS: String) {
```

Comment:
nit: please document the expected format of each of these parameters; IIUC from the example below they're hex-encoded bytes?